catalog/registry.go

```diff
@@ -64,13 +65,13 @@ var (
 // Registrar is a factory for creating Catalog instances, used for registering to use
 // with LoadCatalog.
 type Registrar interface {
-	GetCatalog(catalogURI string, props iceberg.Properties) (Catalog, error)
+	GetCatalog(ctx context.Context, catalogURI string, props iceberg.Properties) (Catalog, error)
```

Comment:
we should update this, the `catalogURI` param should be renamed `catalogName` based on the updated semantics

Reply:
👍